docs/users/basics/gas.md

```diff
@@ -11,36 +11,146 @@ Learn about the differences between `Gas` and `Fees` in Ethereum and Cosmos. {sy
+
+```markdown
+Total Fees = Gas * Gas Price (the price per unit of gas)
+```
+
+If “gas” was measured in kWh, the “gas price” would be the rate (in dollars per kWh) determined by your energy provider, and the “fees” would be your bill. Just as with electricity, gas price is liable to fluctuate over a given day, depending on network traffic.
+
+More on Gas vs. Fees:
+
+- [Cosmos Gas and Fees](https://docs.cosmos.network/main/basics/gas-fees)
+- [Ethereum Gas and Fees](https://ethereum.org/en/developers/docs/gas/)
+
+### 5. How are Fees Handled on Cosmos?
+
+Gas fees on Cosmos are relatively straightforward. As a user, you specify two fields:
+
+1. A `GasLimit` corresponding to an upper bound on execution gas
+2. Either `Fees` or `GasPrice`, which will be used to specify or calculate the transaction fees
+
+The node will entirely consume the fees provided, then begin to execute the transaction. If the `GasLimit` is found to be insufficient during execution, the transaction will fail and roll back any changes made, without refunding the fees provided.
+
+Validators for Cosmos SDK-based chains can specify the `min-transaction-fee` that they will enforce when selecting transactions to include in blocks. Thus, transactions with insufficient fees will encounter delays or fail outright.
+
+At the beginning of each block, fees from the previous block are [allocated to validators and delegators](https://docs.cosmos.network/main/modules/distribution), and they can be withdrawn and spent.
+
+### 6. How are Fees Handled on Ethereum?
+
+Fees on Ethereum include multiple implementations that were introduced over time.
+
+Originally, a user would specify a `GasPrice` and `GasLimit` within a transaction—much like a Cosmos SDK transaction. A block proposer would receive the entire gas fee from each transaction in the block, and they would select transactions to include accordingly.
+
+With proposal EIP-1559 and the London Hard fork, gas calculation changed. The `GasPrice` from above is now split into two separate components: a `BaseFee` and `PriorityFee`. The `BaseFee` is calculated automatically based on the block size and is burned once the block is mined. The `PriorityFee` goes to the proposer and represents a tip, or an incentive for a proposer to include the transaction in a block.
+
+```markdown
+Gas Price = Base Fee + Priority Fee
+```
+
+Within a transaction, users can specify a `max_fee_per_gas` corresponding to the total `GasPrice` and a `max_priority_fee_per_gas` corresponding to a maximum `PriorityFee`, in addition to specifying the `gas_limit` as before. All surplus gas that was not required for execution is refunded to the user.
+
+More on Ethereum Fees:
+
+- [Gas Calculation Docs](https://ethereum.org/en/developers/docs/gas/)
+- [Proposal EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)
+
+## Implementation
```

Comment:
Adding a section about the global MinGasPrices and how it relates to the base fee would be helpful